app/assets/javascripts/admin/templates/components/tags-uploader.hbs

```diff
@@ -0,0 +1,7 @@
+  
+  <label class="btn {{if addDisabled 'disabled'}}">
+    {{d-icon "upload"}}
+    {{i18n 'admin.watched_words.form.upload'}}
+    <input disabled={{addDisabled}} type="file" accept="text/plain,text/csv" style="visibility: hidden; position: absolute;" />
```

Comment:
We should avoid inline styles and move the styling here into a stylesheet.

Reply:
I was following the conventions used [elsewhere](https://github.com/discourse/discourse/search?q=style%3D%22visibility%3A+hidden%3B+position%3A+absolute%3B%22&unscoped_q=style%3D%22visibility%3A+hidden%3B+position%3A+absolute%3B%22). But I agree, and will create a separate PR to clean them all up.

Reply:
PR at #6487